src/forms.ts

```diff
@@ -286,28 +322,22 @@ export default class Form<T> implements IForm<T> {
     let fields: any = {}
     for (let index = 0; index < this.fields.length; index++) {
       const field = this.fields[index]
-      fields[field.name] = field
+      if (field instanceof FormField || field instanceof FormArray) fields[field.name] = field
     }
 
     return fields
   }
-  get fields(): TFormFieldTypeOpts<T>[] {
-    let arr: TFormFieldTypeOpts[] = []
-    for (const fieldName in this.#fields) {
-      arr.push(this.#fields[fieldName])
-    }
-
-    return arr
```

Comment:
@lakardion will this return a list with all fields or formarrays still? or will this just return their values?